app/scripts/services/Citations.svc.js

```diff
@@ -1,28 +1,22 @@
 'use strict';
 
-angular.module('yourStlCourts').factory('Citations', function ($resource,$q) {
-  var CitationResource = $resource('citations/:id');
+angular.module('yourStlCourts').factory('Citations', function ($resource) {
+  var CitationResource = $resource('citations/:id', {id: '@id'});
 
   function find(parameters) {
-    var deferred = $q.defer();
-    CitationResource.get(parameters,function(citations){
+    return CitationResource.get(parameters,function(citations){
```

Comment:
Clark - this is a little weird....

return CitationResouce.get(parameters).$promise.then(.....

Use the promise just like any other consumer. I've never even seen the syntax you're using.